crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -58,7 +58,7 @@ def initialize_state
     end
 
     def load!
-      Crowbar::Lock::LocalBlocking.with_lock(shared: true, logger: @logger) do
+      Crowbar::Lock::LocalBlocking.with_lock(shared: true, logger: @logger, path: lock_path) do
```

Comment:
I'm wondering - shouldn't we pass explicit path only when we realize `Rails` is not available?